### ISSUE TITLE
[Feature request] Better control over theme colors and options

### ISSUE THREAD

**Issue by brneor** [zellij-org/zellij#2160]
As a user, I would like to choose colors of individual elements like active and inactive borders and tabs, background color for selection, and so on. Current implementation of themes only allow me to set the overall color scheme (as far as I'm aware...).

**Comment by brneor** [comment]
@pergamomo yes... The current theme options only allows me to set what color is `green` or `orange`, eg. And then applies these colors on what is defined to be `green` or `orange`.

What I want is to choose what elements will be `green` or `orange` and have more control over this.

**Comment by pergamomo** [comment]
Ah, ok. Yes, I agree that would be a nice to have feature...

**Comment by robclancy** [comment]
it's impossible to make zellij look good because of the obsession with green, the active tab doesn't even look active when there are 2 of them on half the themes because the white stands out more

**Comment by dchersey** [comment]
Heads-up on this one: pane-content selection is actually already theme-driven — since #3242, grid.rs colors the mouse-drag selection from text_selected.base/.background (a color swap, not reverse-video), so setting text_selected in a theme already changes it.

The remaining gap is that it shares the same text_selected component as Zellij's UI "selected text" (the .selected() state of the Text primitive — session-manager rows, strider results, etc.), so you can't color pane-content selection independently of UI selection.

I put up a small, opt-in PR adding an optional pane_selection component that falls back to text_selected when unset (existing themes byte-for-byte unchanged): #5282. Would love your read on whether decoupling is the right direction — since #3242 deliberately unified them — and on naming. Details + open questions in the PR.

**Comment by mfarrugi** [comment]
It would be nice to be able to style pane background color, and this is one of the possible solutions to #2180. 

In tmux I have this to make inactive frames grey.
```
set-option -g window-style 'bg=colour236' 
set-option -g window-active-style 'bg=black'
```
